src/fragments/lib-v1/analytics/js/identifyuser.mdx

```diff
@@ -23,16 +21,13 @@ const attributes = {
     name: "username",
     email: "name@example.com",
     plan: "plan",
-    "phoneNumber": "+11234567890",
-    "age": 25
+    phoneNumber: "+11234567890",
+    age: 25
 }
 
 let userProfile = {
```

Comment:
`const`?

I stole this example directly from the ios docs. I thought I checked that it was compatible with our API. Is there a reason these other attributes don't work?

Reply:
The format of our user profile is slightly different than the other platforms.